src/benchmark/relative_speed.rs

```diff
@@ -8,46 +8,61 @@ pub struct BenchmarkResultWithRelativeSpeed<'a> {
     pub result: &'a BenchmarkResult,
     pub relative_speed: Scalar,
     pub relative_speed_stddev: Option<Scalar>,
-    pub is_fastest: bool,
+    pub is_reference: bool,
+    pub is_faster: bool,
```

Comment:
Maybe rename the latter to
```suggestion
    pub is_reference: bool,
    pub is_faster_than_reference: bool,
```

Thinking about this some more, it's actually not clear what the value of the latter bool should be for the case where `is_reference` is `true`. So maybe introduce an enum instead `pub enum RelativeOrdering { Reference, FasterThanReference, SlowerThanReference }` and use this instead of the two bool ("make illegal states unrepresentable"):
```suggestion
    pub relative_ordering: RelativeOrdering,
```

What do you think?